pylint/reporters/reports_handler_mix_in.py

```diff
@@ -65,7 +74,9 @@ def make_reports(self, stats, old_stats):
                 sect.append(report_sect)
         return sect
 
-    def add_stats(self, **kwargs):
+    def add_stats(  # type: ignore # ReportsHandlerMixIn is always mixed with PyLinter
+        self: "PyLinter", **kwargs
```

Comment:
Can `**kwargs` be typed? I.e. do you know the type for the dict values (in `kwargs`)?

Reply:
Seems to be a mix of `int`, `list`, `dict` and `None`... I guess not? Or I could add a `Union`. What do you prefer?

Reply:
With that mix it might be a case for `Any` 🤔

Reply:
Although `mypy` will probably complain about that in the future... I will add the Union 😄

Reply:
I kinda agree with Marc here, this much type in a Union scream "whatever" or as mypy call it ``Any`` 😄 We'd need to refactor to make this code make sense and we'll probably throw some code if we create a proper class for stats later so we might as well not bother with too much precision in unsalvageable legacy code.

Reply:
> We'd need to refactor to make this code make sense and we'll probably throw some code if we create a proper class for stats later so we might as well not bother with too much precision in unsalvageable legacy code.

Good point! Although imprecise, sometimes `Any` is the better choice after all. Even if only temporarily.

Reply:
Good points! Add `Any` in latest commit!